projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -733,12 +733,8 @@ public static function add_iframed_editor_style() {
 			return;
 		}
 
-		$allowed_pages       = array( 'admin.php', 'themes.php' );
-		$is_site_editor_page = in_array( $pagenow, $allowed_pages, true ) &&
-			isset( $_GET['page'] ) && 'gutenberg-edit-site' === $_GET['page']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
-
-		// WP 5.9 puts the site editor in `site-editor.php` when Gutenberg is not active.
-		if ( 'site-editor.php' !== $pagenow && ! $is_site_editor_page ) {
+		$is_site_editor_page = 'site-editor.php' === $pagenow;
```

Comment:
I think we'll need to keep checking for the old page here if we want this condition to keep working with Gutenberg 13.6 🤔 the problem is that Gutenberg 13.6 will redirect `site-editor.php` to the old `themes.php?page=gutenberg-edit-site`.

Reply:
It looks like the existing code handles both Site editor routes, so you can this leave as is?

Reply:
@creativecoder Are you sure? Unless I'm missing something, if 13.6 is active, accessing `site-editor.php` will redirect to `themes.php?page=gutenberg-edit-site`, so this logic will not apply?

Reply:
Ah, I see what you mean, sorry for the misunderstanding. Yeah, the existing logic does handle both, you're right, though I think the rewritten one is a bit easier to understand.